testing/xsel/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: dai9ah <dai9ah@protonmail.com>
+# Maintainer:
```

Comment:
Are you willing to maintain this software if a new version comes out, or if an issue is encountered?  If so, please add your name and email to the `Maintainer:` line as it is with `Contributor:`.